tst/PortingAssistantExtensionIntegTests/FileUtils.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace PortingAssistantExtensionIntegTests
+{
+    //Copied from PA standalone tests
+    class FileUtils
+    {
+        static string[] allowList = { ".cs", ".json", ".config", ".nupkg", ".targets"};
+        static ISet<string> allowSet = new HashSet<string>(allowList);
+
+        public static bool AreTwoDirectoriesEqual(
+            string dirPath1, string dirPath2)
+        {
+            DirectoryInfo dir1 = new DirectoryInfo(dirPath1);
+            DirectoryInfo dir2 = new DirectoryInfo(dirPath2);
+           
+            // Take a snapshot of the file system.  
+            IEnumerable<FileInfo> list1 = dir1.GetFiles(
+                "*.*", SearchOption.AllDirectories).Where(x => isValidFile(x)).ToList<FileInfo>();
```

Comment:
suggest to sort the two file lists before comparison. I've had situation where test fails when two lists have the same elements but different order.

Reply:
I will handle it. Thanks